drivers/can/stm32_can.c

```diff
@@ -210,13 +210,18 @@ int can_stm32_runtime_configure(struct device *dev, enum can_mode mode,
 	u32_t bs1;
 	u32_t bs2;
 	u32_t sjw;
+	int ret;
 
 	clock = device_get_binding(STM32_CLOCK_CONTROL_NAME);
 	__ASSERT_NO_MSG(clock);
 	hcan.Instance = can;
 
-	clock_control_get_rate(clock, (clock_control_subsys_t *) &cfg->pclken,
-			       &clock_rate);
+  ret = clock_control_get_rate(clock, (clock_control_subsys_t *) &cfg->pclken,
+										 &clock_rate))
+	if (ret) {
+    return ret;
```

Comment:
can you add a LOG_ERR and return -EIO instead of ret please.

Reply:
Sure, I thought about it, but decided to make it simple.
Is that LOG_ERR message suitable? 
LOG_ERR("Failed call clock_control_get_rate: return [%d]", ret);

Reply:
Yes. Thanks for fixing!